public/js/script.js

```diff
@@ -43,65 +42,63 @@ $(document).ready(function() {
     }
     $("#answer-input").prop('disabled', false);
     $('#answer-input').attr('placeholder', 'post an answer');
-    
+
   });
-  
+
   socket.on('new question entered', function(data) {
     socket.emit('get queue');
     console.log('total questions: ' + data);
   });
-  
+
   socket.on('get queue', function(data) {
     if (data.place != 'n/a') {
       $('#question-input').attr('placeholder', 'your question\'s position in queue: ' + data.place + '/' + data.total);
     }
     $('#queue').html('questions in queue: ' + data.total);
   });
-  
+
   socket.on('new answer', function(data) {
-    
     $('#answer-list ul').append(getAnswerSec(data));
   });
-  
+
   socket.on('upvote', function(data) {
     console.log('upvote gotten: ' + data.id);
     $('.score[answer-id="' + data.id + '"][number = "' + data.number + '"]').html(data.score);
     $('.score[answer-id="' + data.id + '"][number = "' + data.number + '"]').parent().parent().parent().attr('score', data.score);
     //sortAnswers();
-    
   });
-  
+
   socket.on('timer', function(data) {
-    if (data.secondsLeft == 0) {
+    if (data.secondsLeft === 0) {
```

Comment:
With JavaScript, I'd highly encourage you to always use triple equals. Double equals will not compare type, and does some odd casting you may not expect.